tests/vcr-request-data/test_consumers.TestFedimgConsumer.test_incompatible_images

```diff
@@ -465,4 +465,84 @@ interactions:
       x-frame-options: [SAMEORIGIN]
       x-xss-protection: [1; mode=block]
     status: {code: 404, message: Not Found}
+- request:
+    body: null
+    headers:
```

Comment:
@sinnykumari this is generated by [vcrpy](https://vcrpy.readthedocs.io/en/latest/) it allows to record http request. The first time the tests are run the requests is recorded into this file. Next time this test will be run it will use this file to mock the request instead of making a real call.

Reply:
Thanks, got it!